### DISCUSSION TITLE
Possible to get cvars of server from before demo began recording?

### DISCUSSION THREAD

**Discussion by Dibberdini** [saul/demofile#371] [Questions] [answered]
Besides the convars.on("change") is it possible to get the cvars of the server (e.g. mp_freezetime, mp_round_restart_delay, mp_roundtime, mp_c4timer) that were set before the demo started recording?

I'm trying to print out these values at the beginning of parsing and notify if any of them get updated.

Thanks!

**Comment by saul** [comment, marked as answer]
`convars.on("change")` is called with `oldValue === undefined` for the initial value of the cvar from when the demo started recording. As for values that were set before the demo started recording, unfortunately that information isn't available.
